package.json

```diff
@@ -9,7 +9,14 @@
     "open:src": "webpack-dev-server --hot --port 4000",
     "build:dev": "webpack --env dev --progress --profile --colors",
     "build:prod": "webpack --env prod --progress --profile --colors",
-    "test": "jest --watchAll"
+    "test": "jest --watchAll",
+    "test:coverage": "npm run test -- --coverage",
+    "cypress:open": "cypress open"
```

Comment:
What do you think about renaming `cypress:open` to `test:e2e`?
In order to keep test commands grouped.